app/viewmodels/TradeJourneyStatusesViewModel.scala

```diff
@@ -36,56 +38,33 @@ object TradeJourneyStatusesViewModel {
   private val notStartedStatus     = "notStarted"
   private val cannotStartYetStatus = "cannotStartYet"
 
-  def buildSummaryList(business: TradesJourneyStatuses, taxYear: Int)(implicit messages: Messages): SummaryList = {
+  def buildSummaryList(tradesJourneyStatuses: TradesJourneyStatuses, taxYear: Int)(implicit messages: Messages): SummaryList = {
```

Comment:
cyclomatic complexity is still too high

![image](https://github.com/hmrc/income-tax-self-employment-frontend/assets/1859969/1338ab4a-6323-4797-a8fb-1a75a7d7c35d)